evap/evaluation/templates/faq.html

```diff
@@ -33,19 +33,20 @@ <h4 id="{{ section.id }}-s">{{ section.title }}</h4>
 
 {% block additional_javascript %}
     <script type="text/javascript">
-        var anchor = window.location.hash.replace("#", "").split('-');
-        id = anchor[0];
-        type = 'q';
-        if(anchor.length > 1)
-            type = anchor[1];
+        const anchor = window.location.hash.replace("#", "").split('-');
+        const id = anchor[0];
+        const type = anchor[1];
 
         if(type == "q" && id != ""){
-            $("#collapse-" + id).collapse('show');
+            const collapse = $("#collapse-" + id);
             window.history.pushState('', id, '/faq#' + id + '-q');
+            collapse.collapse('show');
+            collapse.on('shown.bs.collapse', function () {
+                $("#card-" + id)[0].scrollIntoView({
+                    behavior: "smooth",
+                    block: "center"
+                });
+            });
```

Comment:
how does this behave when clicking on a link with `-s`? the heading might be hidden again by the menubar, could you check?

Reply:
It does indeed hide the heading behind the menu bar. Not removing the code from `base.html` would fix this.

Reply:
yes, please try that out. although i'm wondering whether it is obvious enough which section the link pointed to, since there is no visual hint like there is when selecting a specific question...